### DISCUSSION TITLE
How to edit current PhotoDemon code

### DISCUSSION THREAD

**Discussion by longoaks** [tannerhelland/PhotoDemon#482] [General]
I'm a retired C# developer and would like to be able to work with the PhotoDemon code. I only have Visual Studio 2022 and it appears that editing via GUI does not support VB6 code. What do I need to install in order to load the PhotoDemon code as a project and work with the forms and to debug?

Thanks!
Old Guy....

**Comment by ntstatic** [comment]
search for vb6 cd image online
